common/src/api/http-status-codes.ts

```diff
@@ -0,0 +1,8 @@
+export const HttpStatusCodes = {
+  OK: 200,
+  CREATED: 201,
+  ACCEPTED: 202,
+  NO_CONTENT: 204,
+} as const;
```

Comment:
Are we intending to use this only for 20X codes or do we plan to use this for 30x, 40x, and 50x?

Reply:
We will do so, soon.